plugin/helpers.go

```diff
@@ -42,3 +47,13 @@ type Helpers interface {
 type HelpersImpl struct {
 	API API
 }
+
+func ensureServerVersion(required string, actual string) error {
```

Comment:
Maybe we convert this to method of `HelpersImpl` and don't require the server version as a parameter? That would make the other code easier.
```suggestion
func (p *HelpersImpl) ensureServerVersion(required string) error {
```

Reply:
Yes, it will make the other code easier. As far I under the `HelpersImpl` is available to the plugins and it exposes the helper methods to be used. On the other hand `ensureServerVersion` is an internal method to be used by other helpers method as a utility to check the server version. What do you think?

Reply:
`HelpersImpl` is indeed a available to plugins. But if we make `ensureServerVersion` an unexported method, we get the best of two worlds: We can use it internally and it's not usable my plugins.